src/components/20-molecules/accordion/index.scss

```diff
@@ -1,96 +1,90 @@
-.m-accordion {
+:host {
   display: block;
+}
 
-  &__container {
-    border-bottom: solid 1px $color-silver;
-  }
+:host(:first-of-type) {
+  border-top: 1px solid $color-silver;
+  box-sizing: border-box;
+}
+
+.m-accordion {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1px;
+  border-bottom: solid 1px $color-silver;
+  box-sizing: border-box;
 
   &__title {
     display: block;
-    @include text(t1);
-
-    margin: auto 0;
+    @include text(t1, semibold);
 
     &--small {
       @include text(t3, semibold);
     }
   }
 
-  &__title-container {
-    display: flex;
-    justify-content: space-between;
-    padding: 16px 4px;
-    box-sizing: border-box;
-    cursor: pointer;
-    color: $color-axa-blue;
+  &__icon {
+    margin-right: $spacing-3;
 
-    &--closed {
-      margin: auto 0 auto 12px;
-      width: 24px;
-      height: 24px;
+    svg {
+      width: 48px;
+      height: 48px;
     }
+  }
 
-    &-title {
-      display: flex;
+  &__icon-expand {
+    margin-left: $spacing-3;
+  }
 
-      &-icon {
-        margin: auto 12px auto 0;
-        width: 48px;
-        height: 48px;
+  &__button-flex-wrapper {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
 
-        &--hidden {
-          display: none;
-        }
+  &__button-title-wrapper {
+    display: flex;
+    align-items: center;
+  }
 
-        svg {
-          width: 48px;
-          height: 48px;
-        }
-      }
-    }
+  &__button {
+    display: block;
+    border: none;
+    text-align: left;
+    background-color: $color-prim-white;
+    padding: $spacing-4 $spacing-1;
+    width: 100%;
+    box-sizing: border-box;
+    color: $color-axa-blue;
 
     &:hover {
       background-color: rgba($color-axa-blue, 0.2);
     }
 
-    &:focus {
-      border: 2px solid $color-axa-blue;
-      margin: -2px;
-      outline: none;
-    }
-
+    &:focus,
     &:active {
       background-color: rgba($color-axa-blue, 0.2);
-      border: 2px solid $color-axa-blue;
-      margin: -2px;
+      box-shadow: 0 0 0 2px $color-axa-blue;
```

Comment:
Interesting. I guess it's more subtle this way?

Reply:
yes